### DISCUSSION TITLE
Guide to migration from lizardfs?

### DISCUSSION THREAD

**Discussion by Blackpaw** [moosefs/moosefs#421] [Q&A]
I have a home media server with the data stored on a lizardfs setup 
* lizardfs 3.12
* 6.8 TB Data (data size not including the replication)
* Goal 2 for everything
* 33 TB storage total (9 drives ranging from 3TB to 5TB)
* 3 chunkservers (Running on one server on different ports)

Several questions

1. Can moosefs 3.0. chunkservers use the lizardfs chunks unaltered? I presume the master DB would have to be imported. Would save me a lot of data copying if I could. 
Alternatively I could I just run them simultaneously and copy the data from the lizardfs mount to the moose mount. It would be painfully slow, but I imagine a lot safer.
2. Can I run multiple moosefs chunkservers on the same pc? I do it with lizardfs by customised config files and service scripts to launch them so they run on different TCP ports.

nb. The reason for using lizardfs (and moosefs) is to gain a unified storage pool with redundancy and auto healing. I guess btrfs would sort of serve for this, but I don't trust it.

Thanks.

**Comment by chogata** [comment]
1. At the moment the only way we recommend migrating from LizardFS to MooseFS is by copying files, filesystem to filesystem. There is no MooseFS approved conversion tool.
2. Yes, you can do that, by making customised config files and running the chunkserver processes on different ports. You need to make sure you setup your storage classes in a way that reflects that (so you don't end up with a file that has all copies on the same physical server, in case the whole server gets damaged, not just a single drive with data).

**Reply by Blackpaw** [reply]
Thanks, regards migration I came to the conclusion that an import was not a good idea :)

Started setting up moose on the same server, but ran into apt clashes with lizardfs for moosefs-cgi, so decided to setup lxd containers instead - 3 chunkservers and the master. Passed through the disk mounts from the host. Have a metalogger on a separate physical pc.

Performance is excellent and much cleaner to maintain.

Very pleased, thanks!
